### ISSUE TITLE
关于sc_loss和rec_loss的疑问

### ISSUE THREAD

**Issue by DonlynLee** [wwqq/R3-VAE#2]
作者您好！
仔细研读论文与开源代码后，我有几个问题向您请教：

1. 论文中将 SC 作为衡量语义凝聚力的指标，但代码里实现的 `sc_loss`，是逐层累加量化结果 `quants` 与原始残差之间的余弦相似度构造损失：`sc_loss += (1 - F.cosine_similarity(x, quants, dim=-1).mean()) / self.codebook_layer`
想请问这种损失形式在实际训练中，是否可以有效对应论文定义的 Semantic Cohesion？
2. 针对 Rating quantization 模块，直接选取 Top‑1 （K=1）码字是否能够取得更优效果？（否则训练和推理可能存在gap）
3. 重建损失 `rec_loss` 使用余弦相似度实现。请问在您团队的实验验证中，余弦相似度形式的重建损失相比 MSE 损失，效果是否更好？

期待您的回复，非常感谢！
